components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -181,7 +181,7 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
 
                     return (
                         <AnnouncementBar
-                            showCloseButton={true}
+                            showCloseButton={false}
```

Comment:
Not sure what this announcement bar is for but do we really want an announcement bar that cannot be closed?

Reply:
Just read the ticket, I guess we do

Reply:
Yeah, I guess the thinking is that makes the user open the learn more and read the message in the modal.